canonical/canonical.com
# WD-12230 Created /data/opensearch/what-is-opensearch

## Done
- Created /data/opensearch/what-is-opensearch
- There might still be some input required in design
- New folder created for housing this page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check if all the requirements are met for new page and it is added to sitemap correctly.
- [Copydoc](https://docs.google.com/document/d/1p_dRRGsePAmoBNCZ-7wO6bUAUzxx8Nnr9NJqITMaVzo/edit#heading=h.ve251si43fmo)
- [Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=52-2200&t=4pAQid3eF7qF1WNR-0)
- [Demo](https://canonical-com-1311.demos.haus/data/opensearch/what-is-opensearch)
- Check if all the links are added correctly

## Issue / Card

Fixes # [WD-12230](https://warthogs.atlassian.net/browse/WD-12230)




[WD-12230]: https://warthogs.atlassian.net/browse/WD-12230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
